core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala

```diff
@@ -567,6 +568,105 @@ class BlockManagerSuite extends SparkFunSuite with Matchers with BeforeAndAfterE
     assert(store.getRemoteBytes("list1").isEmpty)
   }
 
+  Seq(
+    StorageLevel(useDisk = true, useMemory = false, deserialized = false),
```

Comment:
Is it hard to write a test to cover the multiple replica behavior?

Reply:
No, it is quite easy. I have added one.